### DISCUSSION TITLE
Need more documentation about 'concurrency' of WorkerOptions

### DISCUSSION THREAD

**Discussion by rosenfl** [taskforcesh/bullmq#344] [General]
Hi 
I did not find any documention about 'concurrency' parameter and other WorkerOptions of BullMQ.
It will create 2 workers listening on the same queue?
Are there side impact? Same message handled twice?
regards
Philippe

**Comment by manast** [comment]
Please let me know if this documentation is enough: https://docs.bullmq.io/guide/workers/concurrency

**Reply by rosenfl** [reply]
thanks
